observations/cifar10.py

```diff
@@ -5,6 +5,7 @@
 import numpy as np
 import os
 import sys
+import six
```

Comment:
Good catch on the `six`. Can you place it above `import sys` to keep imports alphabetized?